pages/sdk-protocol-kit/guides/safe-signatures.md

```diff
@@ -0,0 +1,848 @@
+# Safe Signatures
+
+Understanding and generating signatures can be challenging. In the **Safe{Core} SDK**, we provide a set of utilities that can assist in using signatures with Safe. In this article, we will explain how signatures work and how to generate them using the `@safe-global/protocol-kit` package.
+
+## Setting up the example Safe Account
+
+Your Safe Account can be configured with various thresholds and different owner types. An owner can be any Ethereum address, such as:
+
+- External Owned Account (EOA)
+- Child Signer Safe (A Safe Account that's an owner of another Safe Account)
```

Comment:
That's too specific. Any smart contract that implements an EIP-1271 signature validation method is a valid signer

Reply:
Thanks!, changed